consumer/src/main/resources/application-event.yml

```
@@ -0,0 +1,23 @@
+          group: notification-consumer # 컨슈머 그룹 이름
+          consumer.max-attempts: 2 # 메시지 처리 실패 시 재시도 횟수
+        like-in-0:
+            destination: like
+            content-type: application/json
+            group: notification-consumer
+            consumer.max-attempts: 2
+        follow-in-0:
+            destination: follow
+            content-type: application/json
+            group: notification-consumer
+            consumer.max-attempts: 2
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)